src/analyze.js

```diff
@@ -162,7 +162,7 @@ module.exports = async function (id, code, job) {
     const patternPath = wildcardPath.substr(dirIndex);
     const wildcardPattern = patternPath.replace(wildcardRegEx, (_match, index) => {
       return patternPath[index - 1] === path.sep ? '**/*' : '*';
-    }) || '/**/*';
+    }).replace(/([\/\\]\*\*[\/\\]\*)+/g, '/**/*') || '/**/*';
```

Comment:
Should this regex be in a variable like you did with `wildcardRegEx`?

Reply:
Sure, I've separated that out.